asm/bss_8001C4D0.s

```diff
@@ -1785,7 +1785,7 @@ glabel D_8018E060
 glabel D_8018E0E8
 .skip 40
 
-glabel D_8018E110
+glabel gD_8018E118TotalSize
```

Comment:
I sort of prefer to leave a comment `glabel gTotalSize # D_8018E118 `
But it's not a big deal.

Also, when you compile it creates a .map file. And you can find the location of symbols in there (as long as you have OK)